src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -44,6 +46,16 @@ public void actionPerformed(ActionEvent e) {
 		application.addTest("Figure Joe 1", selectTestFigureOptionListener);
 		application.addTest("Figure Joe 2", selectTestFigureOptionListener2);
 		application.addTest("Figure Jane", e -> { FiguresJane.figureScript(janeAdapter); });
+
+		application.addTest("Rectangle", e -> {
+			ComplexCommand commands = ShapeFactory.rectangle(DriverFeature.getDriverManager().getCurrentDriver());
+			commands.execute();
+		});
+
+		application.addTest("Triangle", e -> {
+			ComplexCommand commands = ShapeFactory.triangle(DriverFeature.getDriverManager().getCurrentDriver());
+			commands.execute();
```

Comment:
It is good to keep a convention used in the original code (class instance instead of lambda)